Packages/vcs/Lib/vcsvtk/pipeline2d.py

```diff
@@ -0,0 +1,194 @@
+from .pipeline import Pipeline
+from .. import vcs2vtk
+
+import vcs
+
+class IPipeline2D(Pipeline):
+    """Interface class for Pipeline2D.
+
+    Defines the virtual method API for subclasses of Pipeline2D.
+
+    This class is an interface. It only contains stubs defining the expected
+    methods for concrete subclasses. Any shared implementations should go into
+    Pipeline2D.
+
+    Internal methods:
+        Documented inline.
+
+    Internal variables:
+        - _resultDict: The 'returned' dictionary from the original
+            VTKPlots.plot2D method. This should be refactored out soon and use
+            a documented API for what this needs.
+        - _gm: The graphics method object.
+        - _template: The vcs template object.
+        - _originalData1: The original data1 object.
+        - _originalData2: The original data2 object.
+        - _data1: The _originalData1 object modified for this pipeline.
+        - _data2: The _originalData2 object modified for this pipeline.
+        - _contourLevels: List of contour levels.
+        - _contourColors: List of contour colors.
+        - _vtkDataSet: The vtkDataSet object with _trimmedData[1|2] set as point
+            or cell scalars.
+        - _vtkGeoTransform: The vtkGeoTransform object associated with this
+            pipeline.
+        - _vtkDataSetBounds: The bounds of _vtkDataSet as
+            tuple(float xMin, float xMax, float yMin, float yMax)
+        - _vtkPolyDataFilter: A vtkAlgorithm that produces a polydata
+            representation of the data.
+        - _colorMap: The vcs colormap object used to color the scalar data.
+        - _useContinents: Whether or not to plot continents.
+        - _dataWrapRange: ??? Need clarification.
+        - _useCellScalars: True if data is applied to cell, false if data is
+            applied to points.
+        - _scalarRange: The range of _data1 as tuple(float min, float max)
+        - _maskedDataMapper: The mapper used to render masked data.
+    """
+
+    def __init__(self, context_):
+        super(IPipeline2D, self).__init__(context_)
+
+        # TODO This should be replaced by getters that retrieve the info needed.
+        self._resultDict = None
+
+        self._gm = None
+        self._template = None
+        self._originalData1 = None
+        self._originalData2 = None
+        self._data1 = None
+        self._data2 = None
+        self._contourLevels = None
+        self._contourColors = None
+        self._vtkDataSet = None
+        self._vtkGeoTransform = None
+        self._vtkDataSetBounds = None
+        self._colorMap = None
+        self._useContinents = None
+        self._dataWrapRange = None # TODO Need some clarification on what this does.
```

Comment:
Updated the documentation to:

`- _dataWrapModulo: Wrap modulo as [YMax, XMax], in degrees. 0 mean 'no wrapping'.`